protocol_driver_grpc_async_callback.cc

```diff
@@ -112,6 +114,8 @@ void ProtocolDriverClientGrpcAsyncCallback::ShutdownClient() {
 namespace {
 class TrafficServiceAsync : public Traffic::ExperimentalCallbackService {
  public:
+  TrafficServiceAsync()
+      : thread_pool_((absl::base_internal::NumCPUs() + 1) / 2) {}
```

Comment:
Maybe explain in a comment this particular choice for the number of threads.

Reply:
I added a comment; may make it configurable later.